### PR TITLE
#82 Enter now logs in user

### PR DESCRIPTION
**Pull request by MartinSpasov**
Simple fix that allows the user to press enter after typing his password to log in

### CODE REVIEW

**Comment by pbgnz** [comment]
Nice, how long did it took you to figure out that bug?

**Comment by MartinSpasov** [comment]
@pbgnz like 3minutes :smile:

File: client/modules/User/components/UserLoginForm/UserLoginForm.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ export class UserLoginForm extends Component {
 
             <div className={styles.center}>
               <button className={styles.btnOutlineSecondary + ' btn btn-outline-secondary ' + styles.logInButton}
-                type="button" onClick={this.loginUser}
+                type="submit" onClick={this.loginUser}
```
Comment (jacobrs) [inline client/modules/User/components/UserLoginForm/UserLoginForm.js]: This is not a valid fix because it creates a bug instead of fixing one.

**Comment by jacobrs** [comment]
Also @MartinSpasov please use the labels, milestones and assignees attributes on the PR's as well.

**Comment by pbgnz** [comment]
Martin try putting a global event listener on the enter key on the login page

**Pull request merged**